rpa-recorder/6.1/modules/ROOT/pages/user-interface.adoc

```diff
@@ -114,6 +114,45 @@ To continue recording, add a Bot Task.
 [NOTE]
 Pause the recording before adding an activity or changing the creation mode of the activities. If you open the pop-up using the respective hotkey, while the recording is running, the recording automatically pauses.
 
+== Web Recording Browser
+
+Use the Web Recording Browser to record website navigation actions. It behaves as a normal web browser and enables you to:
+
+* Select additional elements of a website and determine their XPath and attributes.
```

Comment:
suggestion:
old: "Select additional elements of a website ..."
new "Atomatically identify elements of a website with which user interacts ..."